packages/server/postgres/migrations/1649750150687_GitLabDimensionFieldMap.ts

```diff
@@ -0,0 +1,27 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+  CREATE TABLE IF NOT EXISTS "GitLabDimensionFieldMap" (
+    "id" SERIAL,
+    "teamId" VARCHAR(120) NOT NULL,
+    "dimensionName" VARCHAR(120) NOT NULL,
+    "gid" VARCHAR(140) NOT NULL,
+    "labelTemplate" VARCHAR(100) NOT NULL,
+    PRIMARY KEY ("teamId", "dimensionName", "gid")
```

Comment:
-1 Why have an `id` field if it's not the primary key?  It does not even have a unique constraint.

Reply:
I followed the GitHub example here: https://github.com/ParabolInc/parabol/blob/cf55ed4d0877b1756ffc7213062dde5056ed662b/packages/server/postgres/migrations/1631822263196_GitHubDimensionFieldMap.ts

I'm not sure why. Perhaps @mattkrick could help clarify?

Reply:
every node needs an ID in graphql/relay so i just gave it a one so we didn't have to concat 3 fields together `("teamId", "dimensionName", "nameWithOwner")`. safe to remove if you'd rather concat em

Reply:
I'm also fine with keeping it, but it must have a `unique` constraint since you can just insert/update serial fields.

Reply:
I'm also fine with keeping it, but it must have a `unique` constraint since you can just insert/update serial fields.